astropy/coordinates/tests/test_angles.py

```diff
@@ -863,6 +863,23 @@ def test_repr_latex():
     assert '...' in bigarrangle._repr_latex_()
 
 
+def test_rotation_matrix_deprecation():
```

Comment:
No worries. I see thumbs up above, if this is good to go once given the all green by CI, please add the ``merge-when-ci-passes`` label.